### PR TITLE
 Upgrade flake8 version

### PR DESCRIPTION
**Pull request by Hassan754**

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -10,4 +10,4 @@ pylint
 pytest
 sgqlc
 websockets
--e git+https://github.com/gridsingularity/gsy-framework@feature/D3ASIM-3669#egg=gsy_framework
```
Comment (fievelk) [inline requirements/base.in:13]: Ouch

**Review by fievelk — APPROVED** [review]

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: requirements/base.in (hunk 1 above)
Comment (faizan2590) [reply requirements/base.in:13]: Can you replace branch name with env var for `gsy-e-sdk` tp be tested in `staging` & `production` too?

**Review by faizan2590 — APPROVED** [review]
Left a comment, once addressed LGTM

File: requirements/base.in (hunk 1 above)
Comment (Hassan754) [reply requirements/base.in:13]: @faizan2590 can you elaborate more? do you mean that I should use an environment variable to dynamically select the branch here ?

Comment (faizan2590) [reply requirements/base.in:13]: > do you mean that I should use an environment variable to dynamically select the branch here ?

Yes

Comment (Hassan754) [reply requirements/base.in:13]: Let me try it out, if it's okay I'll work on it after merging this PR if you agree

Comment (faizan2590) [reply requirements/base.in:13]: Sure

Comment (Hassan754) [reply requirements/base.in:13]: It seems that this is not straightforward, kindly have a look at the answers [here](https://www.py4u.net/discuss/173406).
However, I indeed believe that this something to be fixed, I'll create a ticket for it and place it in the upcoming sprint

**Pull request merged**